packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.jsx

```diff
@@ -460,8 +451,7 @@ const AccountTransferForm = ({
                                             onChangeTransferFrom(e);
                                             handleChange(e);
                                             setFieldValue('amount', '');
-                                            setFieldError('amount', '');
-                                            setFieldTouched('amount', false);
+                                            setTimeout(() => setFieldError('amount', ''));
```

Comment:
when the user tries to transfer amount between deriv_fiat and mt5 account, and then switches to mt5 -> deriv_fiat acc, validationAmount function will execute after setFieldError function and we will see "This field is required" error under the amount input, to avoid this issue we wrapped setFieldError into setTimeout to be sure that setFieldError will be executed last